changelogs/server_server/newsfragments/1924.clarification

```diff
@@ -0,0 +1 @@
+In data/api/server-server/definitions/edu.yaml , edu type : content : "m.presence" is changed to "m.typing" as "m.presence" is useless here and not required 
```

Comment:
A small update to the changelog to make it more readable, and to make use of inline code (`) instead of quotations.

If I'm understanding this change correctly, I believe this is the intended reason for the fix?

```suggestion
Update the default EDU example to be `m.typing` rather than `m.presence`, as it is primarily used in the typing notification docs.
```

Reply:
But quotation is used in the syntax not inline code